src/cljsee/parse.clj

```diff
@@ -31,6 +31,12 @@
   (let [form (#'rdr/read* rdr true nil opts pending-forms)]
     (->ReadEval form)))
 
+(defn disabled-read-keyword
+  "Read ::foo/keywords without context"
+  [rdr _ opts pending-forms]
+  (let [form (#'rdr/read* rdr true nil opts pending-forms)]
```

Comment:
This is a more general, possibly unpredictable attempt at reading keywords. For example, if I'm interpreting the implications of this implementation correctly, `::::::[::#{}]` is now a valid keyword. To be on the safer side, could you please rewrite this as a similar implementation to the real `read-keyword`, that simply eliminates the "second `:`" logic?